testutil/testutil.go

```diff
@@ -0,0 +1,9 @@
+package testutil
+
+import "sourcegraph.com/sourcegraph/appdash/internal/wire"
+
+type CollectPackets []*wire.CollectPacket
+
+func (cp CollectPackets) Len() int           { return len(cp) }
+func (cp CollectPackets) Swap(i, j int)      { cp[i], cp[j] = cp[j], cp[i] }
+func (cp CollectPackets) Less(i, j int) bool { return *cp[i].Spanid.Trace < *cp[j].Spanid.Trace }
```

Comment:
I think it'd be better to not create the new `testutil` package and instead just introduce this as a `byTraceID` type within `collector_test.go` as that is the only place it will be used currently.